dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/repositories/DataRequestRepository.kt

```diff
@@ -45,18 +48,21 @@ interface DataRequestRepository : JpaRepository<DataRequestEntity, String> {
             "d.reportingPeriod, " +
             "d.dataRequestCompanyIdentifierType, " +
             "d.dataRequestCompanyIdentifierValue, " +
+            "d.requestStatus, " +
             "COUNT(d.userId))" +
             "FROM DataRequestEntity d " +
             "WHERE (:dataTypes IS NULL OR d.dataTypeName IN :dataTypes) " +
             "  AND (:reportingPeriod IS NULL OR d.reportingPeriod LIKE %:reportingPeriod%)" +
             "  AND (:identifierValue IS NULL OR d.dataRequestCompanyIdentifierValue LIKE %:identifierValue%) " +
+            "  AND (:status IS NULL OR d.requestStatus = :status) " +
```

Comment:
Does it make a difference to use "=" for the status and "LIKE" for identifierValue and reportingPeriod?

Reply:
= tests for equality. LIKE tests for substring matching; it would test for equality as well if there was no %. I think then it would be the same.